lms/envs/common.py

```diff
@@ -2102,12 +2102,6 @@
     'WHAT_IS_VERIFIED_CERT': 'verified-certificate',
 }
 
-######################### VISIBLE SETTINGS ###########################
-# These settings' values will be exposed to all templates
-TEMPLATE_VISIBLE_SETTINGS = [
-    'FEATURES',
-]
-
```

Comment:
Don't we need to reference openedx/stanford/lms/envs/common.py from lms/envs/common.py?